include/l3.h

```diff
@@ -77,7 +77,7 @@ int l3_init(const char *path);
 
   #ifdef L3_LOC_ENABLED
 
-    #define l3_log_simple(msg, arg1, arg2)                              \
+    #define l3_log(msg, arg1, arg2)                                     \
```

Comment:
The core change is simply this renaming. `l3_log()` will, henceforth, be the external logging interface.

Previous commits have already introduced `l3_log_mmap()` as the lower-level interface, so this upper-level renaming falls into place nicely.

Rest of the changes in this change-set are simply refactoring name change.